envconfig.go

```diff
@@ -7,16 +7,38 @@ package envconfig
 import (
 	"errors"
 	"fmt"
+	"io"
+	"os"
+	"path"
 	"reflect"
 	"strconv"
 	"strings"
 	"syscall"
+	"text/tabwriter"
+	"text/template"
 	"time"
 )
 
+const (
```

Comment:
We should probably be using `strconv.ParseBool` anywhere we would be `==` comparing against these (I know there are already some places we're doing this wrong). Separately, these wouldn't need to be public.